From a15c77663133e55e67b77e5c1cf0a5fb504aa99c Mon Sep 17 00:00:00 2001
From: Jamie Gennis <jgennis@google.com>
Date: Thu, 18 Apr 2013 20:36:21 -0700
Subject: [PATCH] trace/events: fix gpu event timestamp formatting

This change fixes the how the gpu_sched_switch timestamp field is formatted.

Signed-off-by: Jamie Gennis <jgennis@google.com>
Change-Id: I273234935254ed15772c9e561c9af20e480004ae
---
 include/trace/events/gpu.h | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/include/trace/events/gpu.h b/include/trace/events/gpu.h
index 09efa71..7e15cdf 100644
--- a/include/trace/events/gpu.h
+++ b/include/trace/events/gpu.h
@@ -11,6 +11,7 @@
 	({ \
 		u64 t = ns + (NSEC_PER_USEC / 2); \
 		do_div(t, NSEC_PER_SEC); \
+		t; \
 	})
 
 #define show_usecs_from_ns(ns) \
@@ -77,7 +78,7 @@ TRACE_EVENT(gpu_sched_switch,
 		__entry->next_job_id = next_job_id;
 	),
 
-	TP_printk("gpu_name=%s ts=%5llu.%06lu next_ctx_id=%lu next_prio=%ld "
+	TP_printk("gpu_name=%s ts=%llu.%06lu next_ctx_id=%lu next_prio=%ld "
 		"next_job_id=%lu",
 		__get_str(gpu_name),
 		(unsigned long long)show_secs_from_ns(__entry->timestamp),
-- 
1.8.0

